test_site/youtube.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+
+
+    <title>DAP test site - YouTube tracking</title>
```

Comment:
I was heading down the road of dynamically creating the page content with puppeteer but I think I like this is a better path for now.  If we end up with like tons and tons of test pages for slightly different setup then it might make sense to change things up to be dynamic

Reply:
> This is a nice idea to have the test pages split by the type of interaction that they are testing.

I agree and I expect the test site will be divided up more as we write more tests. Youtube had to be split into its own page immediately because it loads a bunch of content and thus was messing up the "there are no unexpected requests" tests.